guides/source/caching_with_rails.md

```diff
@@ -512,6 +512,32 @@ class ProductsController < ApplicationController
 end
 ```
 
+Sometimes we want to cache response, for example a static page that never gets
+expired. To achieve that, we can use `http_cache_forever` helper and by doing
+so browser and proxies will cache it indefinitely.
+
+By default cached response will be private, cached only on the user's web
+browser. To make it public response, pass `public: true` parameter to this
```

Comment:
We still are not pulling in the information, that specifying public: true, makes proxies cache them.

This section can be pulled in as is from https://github.com/vipulnsward/rails/blob/5b6eb1d58b48fada298215b2cccda89f993890c3/actionpack/lib/action_controller/metal/conditional_get.rb#L256-L258